email/1.3/modules/ROOT/pages/email-list.adoc

```diff
@@ -1,13 +1,13 @@
 = List Emails with Email Connector Examples - Mule 4
 :page-aliases: connectors::email/email-list.adoc
 
-Anypoint for Email (Email Connector) provides the *List - IMAP* and *List - POP3* operations to list emails from any part of the flow. The following examples illustrate how to configure these operations for the connector.
+Anypoint for Email (Email Connector) provides the *List - IMAP* and *List - POP3* operations to list emails from any part of the flow. The following examples illustrate how to configure these operations for the connector:
 
 * <<send-attachments,List Emails from an IMAPS Server>> +
-Configure the *List - IMAP* operation to list all the emails in the configured IMAP mailbox folder from an IMAPS server with TLS connection. Additionally, set a *For Each* scope and *Set Variable* components to iterate over the emails and save the email body and attachment content in variables.
+Configure the *List - IMAP* operation to list all the emails in the configured IMAP mailbox folder from an IMAPS server that has a TLS connection. Additionally, set a *For Each* scope and *Set Variable* components to iterate over the emails and save the email body and attachment content in variables.
```

Comment:
components > component because the "a" makes it singular. That is why I suggested an alternative wording. Here is another way to fix this:  "Set the set a *For Each* scope and *Set Variable* components..."